### PR TITLE
incremental-backup: update API

### PR DESCRIPTION
**Pull request by danielerez**
Changes proposed in this pull request:
incremental-backup: update API

I confirm that this pull request was submitted according to the [contribution guidelines](https://github.com/oVirt/ovirt-site/blob/master/CONTRIBUTING.md): (please @mention yourself to sign)

This pull request needs review by: (please @mention the reviewer if relevant)
@nirs @rollandf

### CODE REVIEW

File: source/develop/release-management/features/storage/incremental-backup.html.md [hunk 1]
```diff
@@ -199,47 +205,47 @@ Response:
...
+Start incremental backup since checkpoint id "previous-checkpoint-uuid".
 
 Request:
 ```
 POST /vms/vm-uuid/backups
 
-<backup>
-    <incremental>previous-backup-uuid</incremental>
+<vm_backup>
+    <checkpoint>previous-checkpoint-uuid</checkpoint>
```
Comment (nirs) [inline source/develop/release-management/features/storage/incremental-backup.html.md]: ```<checkpoint>``` is confusing. What is wrong with ```<incremental>``` used by libvirt?

```<incremental>``` has the nice properly of telling that the backup is incremental.

File: source/develop/release-management/features/storage/incremental-backup.html.md [hunk 2]
```diff
@@ -199,47 +205,47 @@ Response:
...
-Start incremental backup since backup id "previous-backup-uuid".
+Start incremental backup since checkpoint id "previous-checkpoint-uuid".
 
 Request:
 ```
 POST /vms/vm-uuid/backups
 
-<backup>
-    <incremental>previous-backup-uuid</incremental>
+<vm_backup>
```
Comment (nirs) [inline source/develop/release-management/features/storage/incremental-backup.html.md]: Why vm_backup? We are under "/vms/vm-uuid/backups", so I expect the resource to be "backup".

File: source/develop/release-management/features/storage/incremental-backup.html.md [hunk 3]
```diff
@@ -199,47 +205,47 @@ Response:
...
+    <checkpoint>previous-checkpoint-uuid</checkpoint>
     <disks>
         <disk id="disk-uuid" />
         ...
         ...
     </disks>
-    <status>initiailizing</status>
-    <creation_date>...
-</backup>
+    <phase>initiailizing</phase>
```
Comment (nirs) [inline source/develop/release-management/features/storage/incremental-backup.html.md]: Don't we use ```status``` in other places?

"Wait until backup status is ready" sounds better than "Wait until backup phase is ready"

File: source/develop/release-management/features/storage/incremental-backup.html.md [hunk 4]
```diff
@@ -476,6 +467,13 @@ The backup API should allow specifying some range of checkpoint to
 delete. Probably specifying a checkpoint that should be kept, deleting
 all older snapshots.
 
+### Scratch disk
+
+For now we'll use a scratch disk on the host (created by qemu).
```
Comment (nirs) [inline source/develop/release-management/features/storage/incremental-backup.html.md]: (created by libvirt)

File: source/develop/release-management/features/storage/incremental-backup.html.md (hunk 2 above)
Comment (danielerez) [reply source/develop/release-management/features/storage/incremental-backup.html.md]: It's auto-generated from the restapi entity name. Didn't find a way to easily change it. I'll give it another try..

File: source/develop/release-management/features/storage/incremental-backup.html.md (hunk 1 above)
Comment (danielerez) [reply source/develop/release-management/features/storage/incremental-backup.html.md]: The properties in the rest-api are usually mapped to engine entities. In the engine we only have a notion of checkpoints, and 'incremental' refers to a checkpoint. So it seems more confusing to use 'incremental', It isn't really a property that implies a checkpoint id.. maybe 'incremental_checkpoint' or 'incremental_id'?

File: source/develop/release-management/features/storage/incremental-backup.html.md (hunk 4 above)
Comment (danielerez) [reply source/develop/release-management/features/storage/incremental-backup.html.md]: fixed.

**Pull request merged**